### DISCUSSION TITLE
Visual Studio 2022 Uno extension menu not available

### DISCUSSION THREAD

**Discussion by mbenazechJlsa** [unoplatform/uno#19071] [Q&A]
After install I don't have the Uno menu entry in "Extensions" visual studio menu (I'm trying to sign in).

I checked and Uno platform is in the installed list of extensions.
I tried uninstall/install and still the same (v5.5.53).
In the install log of VSIX installer I got an error (maybe not relevant) :

Failed to load SQL Server Data Tools - Sql Editor$Microsoft.VisualStudio.Data.Tools.SqlEditor. Attempt 0 of 5. Error: Element 'Installation' is missing required child element 'InstallationTarget' or child element value is whitespace.

See the attached log file
[dd_VSIXInstaller_20241213112224_2bac.log](https://github.com/user-attachments/files/18126802/dd_VSIXInstaller_20241213112224_2bac.log)

[original question here](https://marketplace.visualstudio.com/items?itemName=unoplatform.uno-platform-addin-2022&ssr=false#qna)

**Comment by jeromelaban** [comment]
Thanks for the question! The installer log does not reveal anything out of the ordinary.

Would you mind doing the following:
- In `Tools`, `Options`, `Projects and Solutions`, for the `MSBuild project output verbosity`, set it to `Diagnostics`
- Restart Visual Studio
- Open the output window, (`View`, `Output`), then select the `Uno Platform` in the drop down list.
- Open your Uno Platform solution project and rebuild it once .

Can you share this window output in this discussion?

Also, can you create a zip of the project that you created and share it here?

Thanks for your help!

**Comment by mbenazechJlsa** [comment]
Here is the log of the output window : 
```
[VERBOSE] OnSolutionOpened
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[INFO] The Uno.WinUI.DevServer or Uno.UI.DevServer package has not been restored yet, retrying...
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
[VERBOSE] StartDevServerAsync: Waiting for current restore operation
[VERBOSE] Enumerating installed packages
[ERROR] Failed to retrieve the solution's loaded NuGet packages. Make sure a full NuGet restore has completed properly and reload the solution.
```

And here the msbuild log: 
[uno.zip](https://github.com/user-attachments/files/18127277/uno.zip)

**Reply by jeromelaban** [reply]
Thanks for the update, this is very useful. We'll be publishing an additional version of the VSIX that logs some more information, but in the meantime, could you try the following:

- Close visual studio
- Delete the bin/obj folders in the solution
- Open visual studio again
- Post the output of the "Uno Platform" window.

Thanks!

**Comment by yanick76** [comment]
Today, I had a similar issue, but also with an additional message: 'The path to the package Uno.WinUI.DevServer is empty.' There was a red dot on the Hot Reload icon (server unreachable) and a 'devserver isn't able to connect' error in the console. After a series of trials and errors to solve this problem, clearing the NuGet cache finally helped:
`dotnet nuget locals all --clear`
More here:
https://learn.microsoft.com/en-us/nuget/consume-packages/managing-the-global-packages-and-cache-folders

This was probably caused by a version conflict or something similar. I tried the Uno platform several months ago by creating a very similar 'Hello World' project, and that was all I did with Uno. Today, I returned to Uno and ran into this problem.

**Comment by SongOfYouth** [comment]
Do not change the defualt cache path of nuget package!

**Reply by jeromelaban** [reply]
This is not supposed to be a problem, why do you mention this?

**Reply by SongOfYouth** [reply]
This problem bothered me for a long time until I restored the cache path of nuget to its default setting and it disappeared.

**Reply by jeromelaban** [reply]
Let me rephrase, how does the problem manifest itself? If you can give logs or errors so we can enhance this support, it will help.
